Source/Classes/PINRemoteImageManager.m

```diff
@@ -281,6 +282,16 @@ - (void)setValue:(nullable NSString *)value forHTTPHeaderField:(nullable NSStrin
     });
 }
 
+- (void)setRequestConfiguration:(PINRemoteImageManagerRequestConfigurationHandler)configurationBlock {
```

Comment:
I think we *do* need a getter because the property is exposed publicly? Reading the property externally is currently not thread safe unless I'm mistaken.